### PR TITLE
test area calc

### PR DESCRIPTION
**Pull request by rajeeja**
#142

### CODE REVIEW

**Comment by anissa111** [comment]
@rajeeja, can you remind me what this is for again?

**Comment by rajeeja** [comment]
Area calculation didn't have a standalone test, this was mostly to improve the code coverage for testing.

File: test/test_grid.py [hunk 1]
```diff
@@ -196,8 +197,63 @@ def test_integrate(self):
...
+        vgrid = ux.Grid(verts)
+
+        # get node names for each grid object
+        x_var = vgrid.ds_var_names["Mesh2_node_x"]
+        y_var = vgrid.ds_var_names["Mesh2_node_y"]
+        z_var = vgrid.ds_var_names["Mesh2_node_z"]
+
+        vgrid.ds[x_var].attrs["units"] = "m"
+        vgrid.ds[y_var].attrs["units"] = "m"
+        vgrid.ds[z_var].attrs["units"] = "m"
```
Comment (philipc2) [inline test/test_grid.py:216]: Using the variable name dictionary here isn't necessary. 

```Python
vgrid.Mesh2_node_x.attrs["units"] = "m"
vgrid.Mesh2_node_y.attrs["units"] = "m"
vgrid.Mesh2_node_z.attrs["units"] = "m"

File: test/test_grid.py [hunk 2]
```diff
@@ -196,8 +197,63 @@ def test_integrate(self):
...
+    def test_compute_face_areas(self):
+        """Load a grid from file and calculate the area of each face of the
+        mesh."""
+
+        xr_grid = xr.open_dataset(str(mesh_file30))
+        grid = ux.Grid(xr_grid)
+        # by default triangular quadrature is used and order is set to 4th.
+        area = grid.compute_face_areas()
+
+        assert (area.size == grid.ds.nMesh2_face.size)
```
Comment (philipc2) [inline test/test_grid.py:244]:
```Python
assert (area.size == grid.nMesh2_face)

**Review by philipc2 — CHANGES_REQUESTED** [review]
Some small clean-up, otherwise it looks good!

File: test/test_grid.py (hunk 1 above)
Comment (rajeeja) [reply test/test_grid.py:216]: Right, waiting for the new design for updating this old PR.

File: test/test_grid.py (hunk 2 above)
Comment (rajeeja) [reply test/test_grid.py:244]: Right, waiting for the new design for updating this old PR.

Comment (rajeeja) [reply test/test_grid.py:400]: test_grid has gotten really big, we should think about making in more modular in another PR.

**Comment by rajeeja** [comment]
All the new tests in this PR were already merged to main in #285

**Pull request closed without merging**